src/interpreter.cc

```diff
@@ -94,9 +94,9 @@ FuncSignature::FuncSignature(Index param_count,
       result_types(result_types, result_types + result_count) {}
 
 Import::Import() : kind(ExternalKind::Func) {
-  WABT_ZERO_MEMORY(module_name);
-  WABT_ZERO_MEMORY(field_name);
-  WABT_ZERO_MEMORY(func.sig_index);
+  ZeroMemory(module_name);
```

Comment:
These will eventually be `std::string` generally, and be passed via `string_view`. But more refactoring needs to be done before that's possible -- again, due requiring trivial constructors in a few places.